### PR TITLE
Switch to solc v0413

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ env:
     - GETH_URL='https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.6.6-10a45cb5.tar.gz'
     - GETH_VERSION='1.6.6'
     - SOLC_URL='https://github.com/ethereum/solidity/releases/download/v0.4.11/solc-static-linux'
```
Comment (konradkonrad) [inline .travis.yml]: Change this to `https://github.com/ethereum/solidity/releases/download/v0.4.13/solc-static-linux`

Comment (LefterisJP) [inline .travis.yml]: Arghhh .... good catch. Forgot we do that also for solc and not for geth only.

**Comment by konradkonrad** [comment]
Please also change https://github.com/raiden-network/raiden/blob/master/docker/build.Dockerfile#L30 to the latest version!

**Comment by ulope** [comment]
`raiden/Dockerfile` and (the soonish going away) `tools/testnet/files/dockerfiles/raiden/Dockerfile` also still contain URLs that point to `0.4.11`

**Comment by konradkonrad** [comment]
sidenote @ulope: do you think it is feasible to read the solc versions from `.travis.yml` in the dockerfiles ?

**Comment by ulope** [comment]
@konradkonrad I've just been thinking about how we could de-complicate this. We could do it via the Makefile and `ARG`s in the Dockerfile(s)

**Comment by LefterisJP** [comment]
@ulope In any case I adjusted the Docker files in this PR
@konradkonrad fixed travis.yml

**Pull request merged**